src/main/java/kr/codesqaud/cafe/service/UserService.java

```diff
@@ -16,31 +23,49 @@ public UserService(UserRepository userRepository) {
         this.userRepository = userRepository;
     }
 
-    public void signUp(SignUpFormDto dto) {
+    public boolean signUp(SignUpFormDto dto) {
         User user = new User(dto.getUserId(), dto.getPassword(), dto.getName(), dto.getEmail());
         userRepository.save(user);
+        return true;
     }
 
-    public User findById(String id) {
-
-        return userRepository.findById(id);
+    public User findById(String id){
+        return userRepository.findById(id).orElseThrow(()->new NotFoundException("유저를 찾을수 없음"));
     }
 
 
-    public void update(User user, UpdateFormDto dto) {
+    public boolean update(User user, UpdateFormDto dto) {
         User current = user;
-        if (checkPassword(current, dto)) {
+        if (current.checkPassword(dto.getPassword())) {
             userRepository.update(new User(user.getUserId(), dto.getNewPassword(), dto.getName(), dto.getEmail()));
+            return true;
         }
+        throw new NotFoundException("비밀번호입력 오류");
+    }
 
+    public boolean login(User loginUser,String password){
+        if(!loginUser.checkPassword(password)){
+            throw new LoginFailedException("로그인 실패");
+        }
+        return true;
     }
```

Comment:
```suggestion
public boolean login2(User loginUser,String password){
        if(loginUser.checkPassword(password)){
            return true;
        }
        throw new LoginFailedException("로그인 실패");
    }
```

`nit` if 문은 긍정형으로 쓸 수 있도록 노력해주세요.